stories/Molecules/Blocks/Lightbox/Lightbox.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story, canvas } from '@storybook/addon-docs';
+import { Lightbox } from './Lightbox';
+
+<Meta title="Molecules/Blocks/Lightbox" />
```

Comment:
@harishkumar-srijan Component is not placed as per the tree structure...still merging the PR but please correct this one